### DISCUSSION TITLE
Compute and Storage Separation

### DISCUSSION THREAD

**Discussion by thompson0012** [juicedata/juicefs#4278] [General]
i have read the[ introduction of PolarDB ](https://github.com/ApsaraDB/PolarDB-for-PostgreSQL)(Compute Storage Seperation for PostgreSQL)

what i think is: can i use juicedata as the storage layer? (instead of the filesystem)
which means, i store the _**db file**_ from postgres in _**object storage**_ , so that can achieve the same architecture?

postgres -> db file (file system) 
postgres -> db file (juicedata, backed by object storage)

**Comment by thompson0012** [comment]
if it works, it would be solve the problem from scale-out? right?

i would like to hear the why and why not

**Comment by zhijian-pro** [comment]
It is true that some people use juicefs as a storage tier for databases, and we recommend actually testing the performance.
